src/Controller/Component/CalendarComponent.php

```diff
@@ -92,7 +93,16 @@ public function init($year, $month, $span = 10) {
 			throw new NotFoundException('Invalid date');
 		}
 
-		$this->Controller->set('_calendar', ['year' => $this->year, 'month' => $this->month, 'span' => $span]);
+		$viewVars = array_merge(
```

Comment:
For assoc arrays we should use `$new + $defaults`

Reply:
Fixed in d249cf98381db9a3127b7d7903d6a03082893cd0